### DISCUSSION TITLE
not able to read decode message from socket

### DISCUSSION THREAD

**Discussion by kellen0903** [Auties00/WhatsappWeb4j#258] [General]
Hi 

I'm able to send sms and register with mobile API, but not able to read message from socket, the message length is always -1

<img width="874" alt="image" src="https://user-images.githubusercontent.com/17547858/230054353-1cd3f1fc-508b-4b8e-b839-01c4b489857a.png">

**Comment by Auties00** [comment]
Can you try with the latest commit?

**Comment by kellen0903** [comment]
Just tried latest commit, still not working, same issue
